qa/suites/rbd/persistent-writeback-cache/4-pool/cache.yaml

```diff
@@ -3,7 +3,7 @@ overrides:
     conf:
       client:
         rbd_persistent_cache_path: /home/ubuntu/cephtest/write_back_cache
-        rbd_persistent_cache_size: 1073741824
+        rbd_persistent_cache_size: 10737418240
```

Comment:
Is this even going to test the cache? i.e. is this just effectively ensuring that the working set fits within the cache instead of fixing the cache to ensure it writesback data under pressure?

Reply:
> Is this even going to test the cache? i.e. is this just effectively ensuring that the working set fits within the cache instead of fixing the cache to ensure it writesback data under pressure?

Yes, just ensure  the probability of problems is very low. The scene where the cache is very small and the size of the write operation is mixed and the size difference is large.